ieee-quantum-week/bqskit.ipynb

```diff
@@ -0,0 +1,580 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "fig, axs = plt.subplots(1, 2, sharey=True, sharex=True, tight_layout=True)\n",
+    "bins = np.linspace(0, 0.4, 25)\n",
+    "\n",
+    "axs[0].hist(uncompiled_error, bins=bins, label=\"mitigated\", alpha=0.5)\n",
+    "axs[0].hist(uncompiled_unmitigated_error, bins=bins, label=\"unmitigated\", alpha=0.5)\n",
+    "\n",
+    "axs[1].hist(compiled_error, bins=bins, label=\"mitigated\", alpha=0.5)\n",
+    "axs[1].hist(compiled_unmitigated_error, bins=bins, label=\"unmitigated\", alpha=0.5)\n",
+    "\n",
+    "axs[0].set_xlabel('error')\n",
+    "axs[1].set_xlabel('error')\n",
+    "axs[0].set_ylabel('count')\n",
+    "\n",
+    "\n",
+    "axs[0].set_title('Uncompiled')\n",
+    "axs[1].set_title('Compiled')\n",
+    "\n",
+    "axs[0].legend()\n",
+    "axs[1].legend();"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bee6cc06",
+   "metadata": {},
+   "source": [
+    "The plot above shows that when working with compiled circuits, it benefits the user to use error mitigation to obtain more accurate results.\n",
+    "In fact the mean of the data in the \"compiled\" plot decreases, as does the standard deviation when using error mitigation."
```

Comment:
```suggestion
    "In fact, the mean of both histograms decreases when using error mitigation."
```
I would not make strong claims on the standard deviation since the results are probably very problem-dependent and the opposite effect may happen as well.

Reply:
Great point.